tests/test_dbt_project/dbt_project.yml

```diff
@@ -4,9 +4,11 @@ config-version: 2
 
 profile: dbt-sqlite
 
+seed-paths: ["data"]
+
 clean-targets:
   - target
-  - dbt_modules
+  - dbt_packages
```

Comment:
I think I am missing the reason for these changes.

Reply:
dbt v1.0.0 has switched to `dbt_packages` as the default directory for additional packages: https://github.com/dbt-labs/dbt-core/pull/3999

Reply:
and `data-path` is deprecated since v1.0.0 https://github.com/dbt-labs/docs.getdbt.com/blob/63dd291ff53c1930b4aa3a83e0da2c96a2702d98/website/docs/reference/project-configs/seed-paths.md?plain=1#L8